Source/WebCore/rendering/mathml/RenderMathMLFraction.cpp

```diff
@@ -229,7 +229,7 @@ void RenderMathMLFraction::layoutBlock(bool relayoutChildren, LayoutUnit)
         return;
 
     if (!isValid()) {
-        layoutInvalidMarkup(relayoutChildren);
+        RenderMathMLRow::layoutBlock(relayoutChildren);
```

Comment:
IIUC layoutInvalidMarkup is not used anymore, so please just it from RenderMathMLBlock.cpp/h

Reply:
Yep